doc/sphinx/Makefile

```diff
@@ -109,6 +110,10 @@ info:
 	make -C $(BUILDDIR)/texinfo info
 	@echo "makeinfo finished; the Info files are in $(BUILDDIR)/texinfo."
 
+odt: 
+	find ./ -iname "*.rst" -type f -exec sh -c 'pandoc "${0}" -o "${0%.rst}.odt"' {} \;
```

Comment:
pandoc can generate a single ODT file from a list of source files. Can you use the source file ordering used in other single doc formats to generate the single ODT file?

Reply:
It still doesn't render the latex / mathjax formulas; I'm working on another solution

Reply:
recommend you remove this until the other solution is ready